CONTRIBUTING_DOCS.md

```diff
@@ -1,291 +1,5 @@
 # Contributing to Calico documentation
 
-## Overview
+We moved the Calico docs site to [a new docs repository](https://github.com/tigera/docs).
```

Comment:
Do we have an equivalent source of information in the new repo? Do we want to re-use any of this content?

Reply:
We still need to build that out, and I will be reusing some of this content. For now, I just want to direct people to the new repo. In time (not too long!) I'll have the equivalent files built.